models/classes/Translation/Listener/ItemCreatedEventListener.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2024 (original work) Open Assessment Technologies SA.
+ */
+
+declare(strict_types=1);
+
+namespace oat\taoItems\model\Translation\Listener;
+
+use oat\generis\model\data\Ontology;
+use oat\oatbox\event\Event;
+use oat\tao\model\featureFlag\FeatureFlagCheckerInterface;
+use oat\tao\model\TaoOntology;
+use oat\taoItems\model\event\ItemCreatedEvent;
+use Psr\Log\LoggerInterface;
+
+class ItemCreatedEventListener
```

Comment:
- [ ] @shpran It is missing tests, please also check the other new classes. I am not sure all them have tests. Thank you!

Reply:
Done ✅